tests/kernel/run_kselftests.pm

```diff
@@ -84,19 +84,16 @@ sub run
         foreach my $i (@kselftests_suite) {
             install_kselftest_suite($i);
             assert_script_run("cd ./tools/testing/selftests/kselftest_install");
-            #required by the TAP openQA parser
-            assert_script_run("echo t/$i.t .. > $i.tap");
             assert_script_run("./run_kselftest.sh -o $timeout -c $i >> $i.tap", 7200);
```

Comment:
Perhaps out of scope for this simple PR but would it make sense to keep the command output into the serial log? So that if a test break the SUT and it becomes unresponsive we can at least know what happened. I've seen this here for example: https://rmarliere-openqa.qe.prg2.suse.org/tests/1354

Reply:
gonna look into that in the next PR since I need to handle known_issues too, perhaps overall improve the logs collection etc. 
I will look into that there too thx @rbmarliere